### DISCUSSION TITLE
CLI tool to pull data from the API and turn it into markdown files for SSG

### DISCUSSION THREAD

**Discussion by endlessuniverse** [directus/directus#10031] [Feature Requests]
Can you adapt this CLI tool?

> A CLI tool that pulls data from Contentful and turns it into markdown files for Hugo and other static site generators. It also includes an express server that can be used for local development and content previews.

https://github.com/ModiiMedia/contentful-hugo

Thanks!

**Comment by rijkvanzanten** [comment]
> Can you adapt this CLI tool?

Would be an interesting angle, but not something we have bandwidth to take on at the moment. Would you like to lead this effort @endlessuniverse?

For what it's worth, there's a couple really good static site generators out there that don't require the use of Markdown files (exclusively), like Eleventy or Gatsby.

**Reply by endlessuniverse** [reply]
Thank you! I'm going to check out Eleventy.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
